### DISCUSSION TITLE
Open Source version released?

### DISCUSSION THREAD

**Discussion by airsay** [sipstack/sipstack#4] [General]
Was the Sipstack source ever released as promised in July 2021?

**Comment by jonathangeller** [comment]
> Was the Sipstack source ever released as promised in July 2021?

Thank you for your question. Yes, the source was released as promised. To gain access, please request an invite or schedule a demo. Once your request is accepted, we will provide you with access to our private GitHub repository containing the application.

Please note that there are some terms and conditions associated with access. If you have any further questions, feel free to ask!

**Comment by airsay** [comment]
@jonathangeller thanks for the response. I have done as directed.
